modules/common/src/main/java/com/sygic/modules/common/MapFragmentWrapper.kt

```diff
@@ -71,6 +70,8 @@ abstract class MapFragmentWrapper : MapFragment(), SdkInitializationManager.Call
     internal lateinit var permissionManager: PermissionsManager
     @Inject
     internal lateinit var locationManager: LocationManager
+    @Inject
+    internal lateinit var themeManager: ThemeManager
```

Comment:
Takto to být nemůže, setuješ na něm věci ještě před tím, než ho dagger injektuje. Viz inicializace camera a map data modelu.

Reply:
pravda, pravduca